config.json

```diff
@@ -1,34 +1,54 @@
 {
-  "payload_name": "Kian_2",
+  "payload_name": "Gabe_1",
```

Comment:
Does it need to be Gabe_1 if we're only flying one bone?

Reply:
You never explained what the number was for so I just assumed it was just an integer that gets incremented arbitrarily lol